pyproject.toml

```diff
@@ -213,6 +213,7 @@ transformations = [
   "statsmodels<0.15,>=0.12.1",
   'stumpy<1.13,>=1.5.1; python_version < "3.12"',
   'temporian>=0.7.0; python_version < "3.12" and sys_platform != "win32"',
+  'tsbootstrap<0.2,>=0.1.0',
```

Comment:
Can you please explain why are we skipping for all extras (pandas v1 and v2 versions)? It gets skipped from old CI then.

Reply:
Ok, it's changed in the latest commit, resolving.